src/Modal.php

```diff
@@ -217,18 +294,14 @@ public function withoutCloseButton(): self
     }
 
     /**
-     * Returns a new instance with the specified content options.
-     *
-     * @param array $value The content options.
-     *
-     * {@see Html::renderTagAttributes()} for details on how attributes are being rendered.
+     * Returns a new instance with the disabled toggle button.
      *
      * @return self
      */
-    public function contentOptions(array $value): self
+    public function withoutToggleButton(): self
```

Comment:
Majority of options accept boolean value. Should we do it here as well?

Reply:
done.